src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1239,7 +1239,7 @@ public void LangVersion()
             parsedArgs = DefaultParse(new[] { "/langversion:latest", "a.cs" }, _baseDirectory);
             parsedArgs.Errors.Verify();
             Assert.Equal(LanguageVersion.Latest, parsedArgs.ParseOptions.SpecifiedLanguageVersion);
-            Assert.Equal(defaultVersion, parsedArgs.ParseOptions.LanguageVersion);
+            Assert.Equal(LanguageVersion.CSharp7_1, parsedArgs.ParseOptions.LanguageVersion);
```

Comment:
>LanguageVersion.CSharp7_1 [](start = 25, length = 25)

Consider using ```LanguageVersion.Latest``` here so that we wouldn't have to adjust this line every time we bump the latest.